app/views/admin/events/_attendances.html.haml

```diff
@@ -1,18 +1,18 @@
 - invitations.each do |invitation|
-  %ul.no-bullet
+  %ul.list-unstyled.ml-0
     %li
       .row
-        .small-8.medium-8.columns
+        .col-8
           = link_to admin_member_path(invitation.member) do
             = invitation.member.full_name
           - if invitation.member.newbie?
             %p
               %em New attendee!
           %p
             %small= invitation.note
-        .medium-4.small-12.columns
+        .col-4
           - if invitation.verified.blank? && @event.confirmation_required
             = link_to 'Verify', admin_event_invitation_verify_path(@event, invitation), class: 'button round expand tiny sucess', method: :post
-          = link_to 'Cancel attendance', admin_event_invitation_cancel_path(@event, invitation), class: 'button round expand tiny alert', method: :post
+          = link_to 'Cancel attendance', admin_event_invitation_cancel_path(@event, invitation), class: 'btn btn btn-danger', method: :post
```

Comment:
```suggestion
          = link_to 'Cancel attendance', admin_event_invitation_cancel_path(@event, invitation), class: 'btn btn-small btn-danger', method: :post
```

Accidentally added the `btn` class twice. Also, just so the button is not so big can we try using a small button here? They really look huge in the interface 😂.

Reply:
Done, updated the image in the PR too

Reply:
@KimberleyCook nice, I think it looks much better.